lib/crawl.js

```diff
@@ -81,12 +92,16 @@ function crawl(url, flags) {
     // the correct mimetypes/headers
     crawler.on('fetchcomplete', queueItem => {
         if (queueItem.stateData.contentType.indexOf('text/html') > -1) {
-            urls.push({
-                label: queueItem.path,
-                url: queueItem.url,
-                selectors: ['document'],
-                hideSelectors: ['iframe'],
-            });
+            // Clone the default scenario and adjust urls as needed
+            const currentScenario = {...defaultScenario};
+            currentScenario.label = queueItem.path;
+            if( flags.referenceUrl ){
+                currentScenario.referenceUrl = queueItem.url.replace(url, flags.referenceUrl);
+            } else if( Object.prototype.hasOwnProperty.call(currentScenario, "referenceUrl") ){
+                delete currentScenario[referenceUrl];
```

Comment:
`referenceUrl` is undefined. It should either be `currentScenario.referenceUrl` or `currentScenario['referenceUrl']`

Reply:
Thanks! I missed not having the quotes.